composer.json

```diff
@@ -39,27 +39,32 @@
 		"nette/http": "^3.0",
 		"nette/application": "^3.0",
 		"nette/di": "^3.0",
-		"latte/latte": "^2.6",
+		"latte/latte": "^2.6|^3.0",
 		"contributte/api-router": "^4.0"
 	},
 	"require-dev": {
-		"mockery/mockery": "~1.3.4",
+		"mockery/mockery": "^1.3.4",
 		"tracy/tracy": "^2.6",
-		"gamee/php-code-checker-rules": "^2.0",
-		"ninjify/coding-standard": "^0.11.0",
+		"gamee/php-code-checker-rules": "^1.0|^3.0",
```

Comment:
Muzes to vymenit za contributte/code-rules?

Reply:
OK